packages/components/src/components/post-icon/post-icon.tsx

```diff
@@ -113,6 +115,16 @@ export class PostIcon {
 
   // Construct the icon url from different possible sources
   private getUrl() {
+    const file = `${this.name}.svg`;
+
+    if (this.isSSR) {
+      if (this.base) {
+        return `/${this.base}/${file}`.replaceAll(/\/\//g, '/');
```

Comment:
Don‘t merge it like that, this.base could be an absolute url and then the first '/' in the return value would make the it an invalid url!